netaddr.go

```diff
@@ -780,6 +788,43 @@ func (ip IP) string6() string {
 	return string(ret)
 }
 
+// StringExpanded returns the string form of the IP address ip, but IPv6
```

Comment:
Wording nitpicking: I'd write this as "StringExpanded is like String, but <explanation of the IPv6 behavior>". Then there's no need to speak of the v4 and invalid forms being the same, the text makes it clear that everything else is identical to String.